app/livechat/client/views/app/tabbar/visitorForward.js

```diff
@@ -62,18 +67,40 @@ Template.visitorForward.events({
 		} else if (instance.find('#forwardDepartment').value) {
 			transferData.departmentId = instance.find('#forwardDepartment').value;
 		}
+		transferData.userLoggedIn = Meteor.userId();
```

Comment:
- Please, rename the property from `userLoggedIn` to `originalAgentId`;
- You will only use this property if the setting is enabled, so move this line into the correct block of code.